ports/dxcam-cpp/vcpkg.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "dxcam-cpp",
+  "version": "0.2.1",
+  "maintainers": "Fidel Yin <fidel.yin@hotmail.com>",
+  "description": "A high performance screen capturing library for Windows rewriting DXcam in C++.",
+  "homepage": "https://github.com/Fidelxyz/DXCam-CPP",
+  "documentation": "https://dxcam-cpp.readthedocs.io/en/latest/",
+  "license": "MIT",
+  "supports": "windows & !uwp",
+  "dependencies": [
+    "opencv4",
```

Let users opt-out from default features.
```suggestion
    {
      "name": "opencv4",
      "default-features": false
    },
```